### PR TITLE
Tweak Inline Commentary for Installation

### PR DESCRIPTION
**Pull request by zzeleznick**
## Description
Tweaks the inline commentary in the installation instructions

### Checklist

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -74,10 +74,10 @@ You must have Python 3.8 or later installed on the machine where the Semgrep CLI
     1. Install the Semgrep CLI and confirm the installation:
 
         ```console
-        # install
+        # install via pip
```
Comment (s-santillan) [inline docs/getting-started/quickstart.md:77]:
```suggestion
        # install with pip
```

File: docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -106,10 +106,10 @@ You must have Python 3.8 or later installed on the machine where the Semgrep CLI
     1. Using the WSL interface, install the Semgrep CLI and confirm the installation:
 
         ```console
-        #install
+        # install via pip
```
Comment (s-santillan) [inline docs/getting-started/quickstart.md:109]:
```suggestion
        # install with pip
```

**Review by s-santillan — APPROVED** [review]
Minor items only. Feel free to commit then merge.

FYI, "via" (and many latin words/phrases) are best avoided for ESL readers. We use the google style guide, here is its entry on _via_: https://developers.google.com/style/word-list#via

File: docs/getting-started/quickstart.md [hunk 3]
```diff
@@ -39,13 +39,13 @@ You must have Python 3.8 or later installed on the machine where the Semgrep CLI
     1. Install the Semgrep CLI and confirm the installation:
 
         ```console
-        # with homebrew
+        # install with homebrew
         brew install semgrep
 
-        # with pip
+        # install with pip
```
Comment (zzeleznick) [inline docs/getting-started/quickstart.md:45]:
```suggestion
        # install through pip
```

File: docs/getting-started/quickstart.md [hunk 4]
```diff
@@ -39,13 +39,13 @@ You must have Python 3.8 or later installed on the machine where the Semgrep CLI
     1. Install the Semgrep CLI and confirm the installation:
 
         ```console
-        # with homebrew
+        # install with homebrew
```
Comment (zzeleznick) [inline docs/getting-started/quickstart.md:42]:
```suggestion
        # install through homebrew
```

**Pull request merged**